### DISCUSSION TITLE
Ideen: Projekthandbuch

### DISCUSSION THREAD

**Discussion by MatejEvc** [SE-TINF22B2/G3-ApeRepublic#167] [Ideas]
Labels erklären
Meilensteine erklären, evtl ein Graph erstellen
Meetings erklären
Ordnerstruktur zeigen

**Comment by elnaz-gharoon** [comment]
@MatejEvc 
**1-Ziel und Einleitung des Projekts** 
**2- Statistik der Aufwendungen**
2-1 : Gesamtstunden für das Projekt
2-2: Stunden pro Person
2-3: Stunden pro Phase
2-4: Stunden pro Meilenstein

**3- Demo-Bilder des Endprodukts und Entwicklung**

**4- Architektur und Design**
4-1: Technologie und wichtige Designentscheidung
4-2: Datenbankdesign

**5- Anleitung für Benutzer**

**Reply by MatejEvc** [reply]
was ist das?

**Reply by elnaz-gharoon** [reply]
sind meine Idee zum Erweiterug von Handbuch, dachte hier ist Ideesammlung zum Handbuch

**Reply by MatejEvc** [reply]
Hier sind die Sachen die noch evtl. gemacht werden oder gemacht werden müssen, das Projekthandbuch existiert schon
